packages/core/src/types/IViewport.ts

```diff
@@ -110,17 +110,76 @@ export type ReferencedImageRange = ViewReference & {
   referencedImageId: string;
 };
 
+/**
+ *  A referenced plane specifies a planar coordinate system
+ * The most degenerate case has a single point, and includes all planes
+ * which pass through that point.  For this case, if a view plane normal
+ * is required to display the point, the current view plane normal of the
+ * viewport is recommended, or the acquisition plane if there is no view plane
+ * normal.
+ *
+ * A colinear plane restricts the set of all planes containing point to those
+ * which are around the line from point to point+inPlaneVector1
+ * If a view plane normal is required to display the line, the current view up
+ * cross product with the inPlaneVector1 is recommended if these are not colinear.
+ * Otherwise, the cross product of inPlaneVector1 and the current view plane normal
+ * is recommended.
+ *
+ * A specific plane reference further restricts the colinear planes to those
+ * which also include point+inPlaneVector2
+ * The view plane normal to display this point is required to be the cross product
+ * of the two inPlaneVector values.
+ */
+export type ReferencedPlane = {
+  FrameOfReferenceUID: string;
+
+  /** A single point within the reference plane is required for all references */
+  point: Point3;
+
+  /** An inPlaneVector1 is required for all colinear referenced planes.
+   * Shall not be undefined if inPlaneVector2 is defined.
+   */
+  inPlaneVector1?: Point3;
```

Comment:
vec3? instead of Point3?

Reply:
Everything else we declare is Point3, and vec3 is a Point3 in everything except declaration so I'd prefer to go with Point3.